Platform/CoffeelakeBoardPkg/CfgData/CfgDataInt_Cflh.dlt

```diff
@@ -38,6 +38,7 @@ MEMORY_CFG_DATA.UserBd                    | 0x01
 MEMORY_CFG_DATA.DualDpc                   | 0x01
 
 GPU_CFG_DATA.DdiPortFHpd                  | 0x0
+GEN_CFG_DATA.PayloadId                    | 'UEFI'
```

Comment:
Why do we change this default PayloadId ?   It is not related to the UEFI variable.